api/adobe_vendor_id.py

```diff
@@ -801,7 +801,7 @@ def _encode(self, iss=None, sub=None, iat=None, exp=None):
         if exp:
             payload['exp'] = self.numericdate(exp) # Expiration Time
         return base64.encodebytes(
-            jwt.encode(payload, self.secret, algorithm=self.ALGORITHM)
+            jwt.encode(payload, self.secret, algorithm=self.ALGORITHM).encode()
```

Comment:
Did jwt change such that encode now returns a Unicode string?

Reply:
Yes, with [v2.0.0](https://pyjwt.readthedocs.io/en/stable/changelog.html#jwt-encode-return-type) a string is returned instead of a byte string.